inc/modules/export/epub/class-epub201.php

```diff
@@ -2603,10 +2621,18 @@ protected function createNCX( $book_contents, $metadata ) {
 		if ( empty( $this->manifest ) ) {
 			throw new \Exception( '$this->manifest cannot be empty. Did you forget to call $this->createOEBPS() ?' );
 		}
+		$authors = '';
+		if ( isset( $metadata['pb_authors'] ) && is_array( $metadata['pb_authors'] ) && ! empty( $metadata['pb_authors'] ) ) {
+			$authors = [];
+			foreach ( $metadata['pb_authors'] as $author ) {
+				$authors[] = $author['name'];
+			}
+			$authors = oxford_comma( $authors );
+		}
 
 		// Sanitize variables for usage in XML template
 		$vars = [
-			'author' => ! \Pressbooks\Utility\empty_space( $metadata['pb_authors'] ) ? sanitize_xml_attribute( oxford_comma_explode( $metadata['pb_authors'] )[0] ) : '',
+			'author' => sanitize_xml_attribute( $authors ),
```

Comment:
Before we just display the first author listed. Now we are displaying the list of authors.